.github/workflows/test-release.yml

```diff
@@ -69,9 +69,60 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        dist: [wheel, sdist]
+        python-version: ['3.7', '3.11', '3.12']
+        python-architecture: ['x64']
+        extra-args: ['']
+        os: [ubuntu, macos, windows]
+        version-restrict: ['']
+        include:
+          # code-coverage build on macos python 3.9
```

Comment:
I think we can skip the specialized jobs in release testing.
32-bit and version restrict tests need modifications of the requirements files which is not done in this workflow.